extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java

```diff
@@ -210,6 +209,14 @@ private ConfigurationBuilder builderFromProperties(Properties properties) {
             properties.put(ConfigurationProperties.TRUST_STORE_TYPE, infinispanClientRuntimeConfig.trustStoreType.get());
         }
 
+        if (infinispanClientRuntimeConfig.serverList.isPresent()) {
```

Comment:
This is unneeded, this is already done at https://github.com/quarkusio/quarkus/blob/main/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java#L167

Reply:
yeah I don't know what happened here, it's a mistake